lib-multisrc/wpcomics/src/eu/kanade/tachiyomi/multisrc/wpcomics/WPComics.kt

```diff
@@ -20,23 +25,28 @@ import java.util.Locale
 abstract class WPComics(
     override val name: String,
     override val baseUrl: String,
-    override val lang: String,
-    private val dateFormat: SimpleDateFormat = SimpleDateFormat("HH:mm - dd/MM/yyyy Z", Locale.US),
-    private val gmtOffset: String? = "+0500",
+    final override val lang: String,
+    protected val dateFormat: SimpleDateFormat = SimpleDateFormat("HH:mm - dd/MM/yyyy Z", Locale.US),
+    protected val gmtOffset: String? = "+0500",
 ) : ParsedHttpSource() {
 
     override val supportsLatest = true
 
     override val client: OkHttpClient = network.cloudflareClient
 
     override fun headersBuilder(): Headers.Builder = Headers.Builder()
-        .add("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:84.0) Gecko/20100101 Firefox/84.0")
         .add("Referer", baseUrl)
```

Comment:
use `super.headersBuilder()` to keep default useragent

also referer should be "$baseUrl/" since regular browsers send the trailing slash